packages/core/src/specs/LottieAnimationViewNativeComponent.ts

```diff
@@ -27,9 +37,11 @@ export interface NativeProps extends ViewProps {
   enableMergePathsAndroidForKitKatAndAbove?: boolean;
   hardwareAccelerationAndroid?: boolean;
   cacheComposition?: boolean;
-  colorFilters?: ReadonlyArray<string>;
+  colorFilters?: ReadonlyArray<ColorFilterStruct>;
```

Comment:
I just realized that `LottieView.tsx` converts `colorFilters` and `textFilters` into `string[]`. I wonder if we should get rid of the extra serialization

Reply:
I gave that a try but it didn't work for me. It might be solved with a newer RN version but the last time I checked the Android build failed to generate non-broken code via that type spec definition.

Reply:
But on android codegen will just generate `ReadableArray`, right? @alfonsocj